pinc/maybe_mail.inc

```diff
@@ -50,15 +50,13 @@ function maybe_mail( $to, $subject, $message, $additional_headers = null )
 }
 
 function maybe_mail_project_manager( $project, $info, $prefix)
-// $project is either a string containing a projectid,
-// or an associative array containing various attributes of a project.
 {
-    global $code_url, $site_name, $site_signoff;
+    global $code_url, $site_name, $site_signoff, $site_abbreviation;
 
-    $project = new Project($project);
     $projectid = $project->projectid;
     $nameofwork = $project->nameofwork;
     $username = $project->username;
+    configure_gettext_for_user($username);
 
     $body =
 _("Hello")." $username,\n".
```

Comment:
Let's make this `sprintf(_("Hello %s,"), $username)` as that maps to an already-translated string.

Reply:
The "Regarding" bit below is going to be a really annoying thing for translators to translate. I will defer to @srjfoo, but we might want to use:
```
" . sprintf(_("Regarding project \"%s\""), $nameofwork) . ":
```